LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/Region.cs

```diff
@@ -15,6 +15,8 @@ public abstract class Region
     {
         private const ushort MAX_RX_DELAY = 15;
 
+        protected const double EPSILON = 0.00001;
```

Comment:
Why do we have this tolerance? I never saw the frequencies having some delta with plain values from the packet forwarder/basic station onward.

Reply:
I thought this would be safer than just comparing the equality of two doubles, here is what I found in the official docs https://docs.microsoft.com/en-us/dotnet/api/system.double.equals?redirectedfrom=MSDN&view=net-6.0#System_Double_Equals_System_Double_ where using a threshold seems to be recommended when comparing doubles. In this case it might be indeed not required since we are not doing any arithmetic operations on the frequencies so I'm opening to reverting this - let me know.